x-pack/legacy/plugins/apm/public/context/MatchedRouteContext.tsx

```diff
@@ -4,11 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 import React, { useMemo, ReactChild } from 'react';
-import { matchPath } from 'react-router-dom';
+import { matchPath, match } from 'react-router-dom';
```

Comment:
let's do `match as Match`, to make it clear that it's a TS interface/type. Not sure why they're exporting it as lowercase, but I don't see a particularly good reason to extend that into our own code.